src/core/src/core_logic/PatchInstaller.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.lifecycle_manager = lifecycle_manager
 
         self.package_manager = package_manager
+        self.package_manager_name = self.package_manager.__class__.__name__
```

Comment:
Non-blocker: You're getting the class name here. The actual package manager name is obtained by: self.package_manager.get_package_manager_setting(Constants.PKG_MGR_SETTING_IDENTITY)

which will return things like Constants.APT, etc.

Reply:
Thanks, I was not aware of this. I will take care of this in a separate PR. Completing this PR.